hazelcast/src/test/java/com/hazelcast/config/MapStoreConfigTest.java

```diff
@@ -12,24 +12,45 @@
 import java.util.Map;
 import java.util.Properties;
 
+import static com.hazelcast.config.MapStoreConfig.DEFAULT_WRITE_BATCH_SIZE;
+import static com.hazelcast.config.MapStoreConfig.DEFAULT_WRITE_DELAY_SECONDS;
 import static com.hazelcast.config.MapStoreConfig.InitialLoadMode.EAGER;
 import static com.hazelcast.config.MapStoreConfig.InitialLoadMode.LAZY;
 import static org.junit.Assert.*;
 
 /**
  * Test MapStoreConfig
  */
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelTest.class})
+//@RunWith(HazelcastParallelClassRunner.class)
```

Comment:
I think these two lines should be enabled again, so the test is picked up by the correct Jenkins build.